### ISSUE TITLE
Phenylephrine-induced blanching of eye redness

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9888]
**Preferred term label:**
Phenylephrine-induced blanching of eye redness

**Synonyms**

**Definition (free text, please give PubMed ID)**
Instillation of 2.5% phenylephrine drops induces blanching of vessels of conjunctival and/or episcleral vessels, but not scleral vessels

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Episcleritis

**Your nano-attribution (ORCID)**

**Comment by MickeySegal** [comment]
On second thought it makes sense to reverse the finding, since the blanching is what one would see commonly from any conjunctivitis.  
Term Label: Phenylephrine does not induce blanching of eye redness
Definition: Instillation of 2.5% phenylephrine drops into the eye does not induce blanching of eye redness; failure to blanch suggests redness caused by scleral vessel engorgement (e.g., scleritis) or extravascular cause (e.g., hemorrhage), not episcleritis or conjunctivitis.
Diseases: scleritis

**Comment by pnrobinson** [comment]
@MickeySegal 
added as suggested with 
Phenylephrine does not induce blanching of eye redness
HP:0034983
